contracts/badger-bridge/BadgerBridgeAdapter.sol

```diff
@@ -163,6 +176,20 @@ contract BadgerBridgeAdapter is OwnableUpgradeable, ReentrancyGuardUpgradeable {
         uint256 startBalanceRenBTC = renBTC.balanceOf(address(this));
         uint256 startBalanceWBTC = wBTC.balanceOf(address(this));
 
+        if (_token == address(ibBTC)) {
+            //redeem ibbtc
+            ibBTC.safeTransferFrom(msg.sender, address(this), _amount);
+            uint256 poolId = vaultToPoolId[_vault];
+            uint256 redeemAmount = 0;
+            if (poolId == 3) {
+                redeemAmount = yearnWbtcPeak.redeem(_amount);
+            } else {
+                redeemAmount = settPeak.redeem(poolId, _amount);
+            }
+            IERC20(_vault).safeTransfer(msg.sender, redeemAmount);
```

Comment:
Would modify the logic to not transfer the tokens back and forth. Will probably need to refactor the conditional logic here so it's cleaner. 
Ideally we just transfer the initial token from the user one time and then unwrap to renbtc based on a series of conditionals and burn via the renvm gateway.